app/src/lib/remote-parsing.ts

```diff
@@ -19,10 +19,11 @@ interface IGitRemoteURL {
 export function parseRemote(url: string): IGitRemoteURL | null {
   // Examples:
   // https://github.com/octocat/Hello-World.git
+  // https://github.com/octocat/Hello-World.git/
   // git@github.com:octocat/Hello-World.git
   // git:github.com/octocat/Hello-World.git
   const regexes = [
-    new RegExp('^https?://(.+)/(.+)/(.+?)(?:\.git)?$'),
+    new RegExp('^https?://(.+)/(.+)/(.+?)(?:\.git/?)?$'),
```

Comment:
This was added because I saw a log files where someone had their remote url set to the full url with a trailing slash. I suspect that URL was either added manually or by another tool but since GitHub happily supports it I figured we should as well.